control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator.go

```diff
@@ -249,6 +252,13 @@ func ReconcileDeployment(dep *appsv1.Deployment, params Params, apiPort *int32)
 		})
 	}
 
+	if os.Getenv(rhobsmonitoring.EnvironmentVariable) == "1" {
```

Comment:
why does it need to pass it down here?

Reply:
The CNO contributes 2 service monitors to the control plane namespace. We will need to have the CNO react to the setting just like the CPO. This sets it up so that we can make the change to the CNO as a follow up.